drake/examples/Atlas/urdf/atlas_minimal_contact.urdf

```diff
@@ -131,6 +131,7 @@
       <inertia ixx="0.000388" ixy="0" ixz="0" iyy="0.000477" iyz="0" izz="0.000379"/>
     </inertial>
   </link>
+  <frame link="l_hand" name="l_hand_mount" rpy="0 -1.5707963267948966 3.1415926535897" xyz="0 -0.195 0"/>
```

Comment:
This frame replaces the atlas-version-specific offsets stored in Atlas.m. Unfortunately, we can't delete those offsets from the matlab implementation yet, since they're used in constructing the model for sim.